web/src/pages/index.page.tsx

```diff
@@ -50,8 +54,30 @@ const Home: NextPage = () => {
 
       <section id="how-it-works">
         <StyledBox>
-          <Typography variant="h4">How it works</Typography>
-          <YoutubeEmbed embedId="yM8RrwQWeJc" />
+          <Box sx={{ height: 600, width: "70%" }}>
+            <Typography variant="h4">How it works</Typography>
+            <Carousel autoPlay={false}>
```

Comment:
Thoughts on making the nav buttons always visible? I think it'd be nicer for switching around

Reply:
I think that's a good call.